packages/esm-service-queues-app/src/queue-table/queue-table-metrics-card.component.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import classNames from 'classnames';
+import { useTranslation } from 'react-i18next';
+import { Layer, Tile } from '@carbon/react';
+import { ConfigurableLink } from '@openmrs/esm-framework';
+import styles from '../patient-queue-metrics/metrics-card.scss';
+
+interface QueueTableMetricsCardProps {
+  value: number | string;
+  headerLabel: string;
+  children?: React.ReactNode;
+}
+
```

Comment:
The style of card already present on the service-queues home page should serve as the example of how this should be styled.